package/gluon-mesh-vpn-tunneldigger/luasrc/usr/bin/tunneldigger-watchdog

```diff
@@ -33,7 +33,9 @@ local function has_mesh_vpn_neighbours()
 end
 
 if uci:get_bool('tunneldigger', 'mesh_vpn', 'enabled') then
-	if io.popen('pgrep -x /usr/bin/tunneldigger'):read('*l') ~= read_pid_file() then
+-- tunneldigger misses .pid file for this script - workaround and simply generate one from process-list
+        os.execute('logger -t tunneldigger-watchdog "generating missing .pid file for gluon...";ps | grep "bin/tunneldigger -f" -m 1 | cut -d " " -f 2 >/var/run/tunneldigger.mesh-vpn.pid')
+        if io.popen('pgrep -x /usr/bin/tunneldigger'):read('*l') ~= read_pid_file() then
 		os.execute('logger -t tunneldigger-watchdog "Process-Pid does not match with pid-File."')
 		restart_tunneldigger()
 		return
```

Comment:
Let's just drop this whole condition (and the function `read_pid_file`) - procd will automatically restart tunneldigger anyways, so it is not needed anymore.

At some point, we could also replace the `uci:get_bool('tunneldigger', 'mesh_vpn', 'enabled')` with a query to procd whether the service is actually supposed to be running (so manually starting and stopping it will be noticed by the watchdog), but that's fairly low priority, so let's leave that for a future PR.